### PR TITLE
Add ability to edit Allocation Helper text

### LINKED ISSUE #1568: Add ability to edit Allocation Helper text
**Summary:**

1. **What is actually happening:**
Admin cannot edit allocation helper text
2. **What should be happening:**
Admin should be able to edit Allocation Helper text

This is the same issue as #1566 but for allocations. 

This text should be able to be rendered in **Markdown**

3. **Screenshots/GIF/Video recording**

### PR DESCRIPTION
**Pull request by MooSeMoos**
## Motivation and Context
resolves #1568
## Description

Admin cannot edit allocation helper text 

## Test and Deployment Plan

go to Allocate page--> Give --> click edit help text and change text-->click save and check saved text

## Screenshots (if appropriate):
Screenshot from 2022-11-15 18-49-40
Screenshot from 2022-11-15 18-49-21
## Reviewers
@reeserj
@sshmm

## Related Issue
#1568

### CODE REVIEW

File: src/pages/GivePage/index.tsx [hunk 1]
```diff
@@ -17,24 +21,40 @@ import { IEpoch, IMyUser } from '../../types';
...
+  PotentialTeammate,
+  QUERY_KEY_CIRCLE_ALLOCATION_TEXT,
+} from './queries';
 
 export type Gift = Awaited<ReturnType<typeof getPendingGiftsFrom>>[number];
 
 // the amount of time to wait for inactivity to save
+const schema = z.object({
+  alloc_text: z
+
```
Comment (sshmm) [inline src/pages/GivePage/index.tsx:43]: could you remove unnecessary spaces?

File: src/pages/GivePage/index.tsx [hunk 2]
```diff
@@ -17,24 +21,40 @@ import { IEpoch, IMyUser } from '../../types';
...
+});
+type allocationTextSchema = z.infer<typeof schema>;
 const SAVE_BUFFER_PERIOD = 1000;
 
 const GivePage = () => {
   const address = useConnectedAddress();
   const {
     circle: selectedCircle,
-    myUser,
+    myUser: me,
```
Comment (sshmm) [inline src/pages/GivePage/index.tsx:57]: is this change necessary? you can use myUser directly to reduce changes.
you will reduce the changes alot if you kept myUser

**Review by sshmm — CHANGES_REQUESTED** [review]

File: src/pages/GivePage/index.tsx (hunk 1 above)
Comment (MooSeMoos) [reply src/pages/GivePage/index.tsx]: done

File: src/pages/GivePage/index.tsx (hunk 2 above)
Comment (MooSeMoos) [reply src/pages/GivePage/index.tsx]: done

**Review by sshmm — COMMENTED** [review]
@wingfeatherwave  I finished my review. you can take it from here

**Comment by wingfeatherwave** [comment]
Updating UI.  minor enhancements for contributions help text editing as well.  mobile fixes.

Screenshot 2022-11-18 at 10 02 26 AM
Screenshot 2022-11-18 at 10 02 49 AM

**Review by wingfeatherwave — APPROVED** [review]
Nice work!  I added some ui changes.
